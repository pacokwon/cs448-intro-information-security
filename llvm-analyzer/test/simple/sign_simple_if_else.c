#include "homework.h"

int main()
{
  int x = source();

  if (x >= 0) {
    int y = 20 / x;
    }
    else
    {
        int z = 30 / x;
    }
}
