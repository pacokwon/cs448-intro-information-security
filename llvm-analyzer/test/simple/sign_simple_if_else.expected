Potential Division-by-zero @ simple/sign_simple_if_else.c:main:8:16, %call = Top
